shared/studio/components/databasePage/databasePage.module.scss

```diff
@@ -21,6 +21,15 @@
     grid-template-columns: 1fr;
     padding: 0;
   }
+
+  @include breakpoint(mobile) {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 10;
+  }
```

Comment:
I'm not sure what this is meant to do? It ends up covering the header nav: 
![image](https://github.com/edgedb/edgedb-ui/assets/2085287/f890f534-c42e-41e3-a644-109a15f0bcf2)

Reply:
fix is coming : )